src/olympia/addons/serializers.py

```diff
@@ -870,24 +872,31 @@ def get_apps(self, obj):
         return [app.short for app in obj.approved_applications]
 
 
-class AddonSerializer(serializers.ModelSerializer):
+class AddonSerializer(AMOModelSerializer):
     authors = UserSerializerWithPictureUrl(
         many=True, source='listed_authors', read_only=True
     )
     categories = CategoriesSerializerField(source='all_categories', required=False)
     contributions_url = ContributionSerializerField(
-        source='contributions', required=False
+        source='contributions',
+        required=False,
```

Comment:
no changes here too...

Reply:
I see a few more changes in this file that you can revert but I won't comment on each line.